.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,123 @@
+name: Bug Report
+description: Create a report to help us improve
+title: "[Bug]: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        A clear and concise description of what the bug is.
+
+        > *__Note:__* Please do not submit support requests or "How to" questions here. Instead, please use our [Discussion forum](https://github.com/openvinotoolkit/anomalib/discussions/categories/q-a)
```

Comment:
I think this is supposed be editable as a text field? It doesn't show up that way when I try it on your fork.